core/controllers/base.py

```diff
@@ -398,6 +398,11 @@ def render_template(
 
         self.response.expires = 'Mon, 01 Jan 1990 00:00:00 GMT'
         self.response.pragma = 'no-cache'
+
+        # cache_slug passed to the template should not have trailing or ending
```

Comment:
Should I add the test in utils_test.py, since a method in utils.py returns the cache_slug?